app/assets/javascripts/application.js

```diff
@@ -36,6 +36,10 @@ function onContentLoad() {
   // highlight tabs with errors
   tfm.tools.highlightTabErrors();
 
+  $('body').popover({selector: 'a[rel="popover"]'});
+  tfm.tools.activateTooltips();
+  tfm.tools.activateDatatables();
```

Comment:
These were mistakenly moved into `highlightTabErrors` when it was extracted from this file.